### ISSUE TITLE
Java Error with Quiver

### ISSUE THREAD

**Issue by Calebplaysgames** [DeflatedPickle/Quiver#177]
**Describe the bug**

**To Reproduce**
Steps to reproduce the behaviour:
1. Litterally open the app

**Expected behaviour**

It should open.

**Screenshots**
![nope](https://user-images.githubusercontent.com/53958544/171487826-0ab6c619-0f7a-4ebf-b2a0-99194a7a8ac9.PNG)

**Desktop (please complete the following information):**
 - OS: Windows
 - Version 10

**Additional context**

**Comment by Calebplaysgames** [comment]
note: Yes Im using the exact linked java 8 version.

**Comment by DeflatedPickle** [comment]
Are you sure? Do you have multiple versions of Java installed? I think you can check with `java --version`

**Comment by Calebplaysgames** [comment]
> 

nope, the minecraft form for texture pack creator programs says that quiver just has issues with newer windows 10 versions.

**Comment by DeflatedPickle** [comment]
Ah, wonderful. I won't be debugging this as I only use Linux but if anyone else wants to figure it out and submit a PR, that'd be appreciated!

**Comment by PolskiKocurek** [comment]
i have too this error and i have multiple java versions installed
